src/components/frontpage/Countdown.astro

```diff
@@ -0,0 +1,45 @@
+---
+const { when } = Astro.props;
+---
+
+<script define:vars={{ when }}>
+  const countdownElement = document.currentScript.parentElement;
+  let countdown = { days: 0, hours: 0, minutes: 0, seconds: 0 };
+  const targetDate = new Date(when).getTime();
+
+  function updateCountdown() {
+    const now = new Date().getTime();
+    const distance = Math.max(0, targetDate - now);
+
+    countdown.days = Math.floor(distance / (1000 * 60 * 60 * 24));
+    countdown.hours = Math.floor(
+      (distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60),
+    );
+    countdown.minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+    countdown.seconds = Math.floor((distance % (1000 * 60)) / 1000);
+
+    countdownElement.querySelectorAll("[data-countdown]").forEach((el) => {
+      el.textContent = countdown[el.dataset.countdown];
+    });
+  }
+
+  setInterval(updateCountdown, 1000);
+  updateCountdown();
+</script>z
+
+<div
+  class="flex gap-8 text-center text-white bg-gradient-to-br from-gray-800 to-gray-900 p-8 rounded-2xl w-fit mx-auto shadow-2xl"
+>
+  <div class="px-6">
+    <p class="text-5xl font-bold tracking-tight" data-countdown="hours">0</p>
```

Comment:
Could we find a way to avoid flash of "0 0 0" before things start up? We should be able to assume (for SSR part if we need it) that current timezone is Oslo Norway.